### ISSUE TITLE
Style the Form, Signup, Signin Components 

### ISSUE THREAD

**Issue by ezioda004** [UdacityFrontEndScholarship/quiz-app#33]
## Style the Form, Signup and Signin Component as shown in this picture.
![image](https://user-images.githubusercontent.com/20020273/38775935-f9971c9a-40ab-11e8-9399-549a88bc98d2.png)

Special note on this one.
You'll have to style in 3 CSS files because we have 3 components. 
1. Signup Component - contains Signup form
2. Signin Component - contains Signin form
3. Forms Component - renders and contains both forms, also switches between Signup/Signin forms on click.

All 3 CSS files are found respectively in their Component folder, so you only need to navigate to the component folder and open their CSS files respectively to get started.
quiz-app => src => Component =>Signup => Signup.js and Signup.css
quiz-app => src => Component =>Signin => Signin.js and Signin.css
quiz-app => src => Component =>Form => Form.js and Signup.css

How to get started? Read our [project setup instructions](https://github.com/UdacityFrontEndScholarship/quiz-app/blob/master/CONTRIBUTING.md#project-setup-instructions).
Fork the project and setup as posted above.

### All you need to do is write normal CSS inside the CSS files mentioned above.
We are using Bootstrap-4 for this project but feel free to use native CSS wherever required.

Note:

1. If you use any self closing tags such as `<img>`, `<input>`, `<br>`
2. Make sure you always close them like this `<img />`, `<input />`, `<br />`

If you add a class to an element example
```
<div  class = "nav navbar"> 
    <!-- rest of your code -->
</div>
```
Make sure you use className instead of class. Example
```
<div  className = "nav navbar"> 
    <!-- rest of your code -->
</div>
````
If you have any doubts regarding this issue, feel free to post it here or contact us on slack.

Please comment on this issue and I'll assign it to you.

**Comment by abhishekakade** [comment]
@ezioda004 sure! Thank you! :)

**Comment by pben1984** [comment]
What does %25 mean in box shadow?

**Comment by abhishekakade** [comment]
@pben1984 `%25` is used in URL encoding to replace empty space. I'm not sure about it's use in box shadow though.

**Comment by pben1984** [comment]
@abhishekakade in image above it says shadow #000000 25%

**Comment by Ajayjam97** [comment]
@pben1984  I think it is the alpha transparency value of #000000.

**Comment by Inderji** [comment]
I have to download this part to work on it??

**Comment by abhishekakade** [comment]
@ezioda004 made a pull request. Please check if any changes are required.

**Comment by abhishekakade** [comment]
@Inderji this issue was assigned to me. You can keep an eye on issues section of this repository for any new issues and ask the mods if you can work on it or if it has already been assigned to someone else :)

**Comment by kadumuri1994** [comment]
@ezioda004 , in the signup form, upon clicking on 'sign in with google/github' I see it is asking to fill the first name, last name user id and password. Is it a valid workflow? If not is this issue being handled by any PR's?

**Comment by ezioda004** [comment]
@kadumuri1994 No, its not valid workflow. We havent setup authentication yet, thats why its acting this way and no, its not being worked by anyone at the moment.

**Comment by abhishekakade** [comment]
Thank you @ezioda004 :raised_hands:
